uber/site_sections/summary.py

```diff
@@ -175,12 +175,115 @@ def food_eligible(self, session):
         }
         return render('summary/food_eligible.xml', {'attendees': eligible})
 
+    def csv_import(self, message='', all_instances=None, form_link=None):
```

Comment:
> I suppose there's no way to make a specific function/page more restrictive than the class?

There isn't a decorator specifically for that, but if you have something like

```
@all_renderable(c.PEOPLE)
class Root:
    def foo(...): ...

    def bar(...): ...
    bar.restricted = [c.ACCOUNTS]
```

this would cause ``foo`` to have the default access level, but ``bar`` disregards the default and replaces it with ``c.ACCOUNTS`` access.

This feature was implemented years ago but I'm not sure we've ever actually used it anywhere, which is why we don't have a separate decorator for this like ``@require_access(c.ACCOUNTS)`` to apply to the function.  If we start doing that a lot, we should definitely implement that decorator.